### DISCUSSION TITLE
A Warning and an Error via LSP

### DISCUSSION THREAD

**Discussion by tevaughan** [zk-org/zk-nvim#121] [Help]
I have noticed two different annoyances, and I'm wondering if there be an easy fix for each.

1. I have a `readme.md` file in each directory in my tree. I also have a directory `doc/md` under which all of my notes generated with an automatic filename via ZkNew are stored. In `doc/md/readme.md` is a link that points back up to `../../readme.md`, the top-level `readme.me`. Whenever I look at this file in neovim, LSP shows a warning, `Ambiguous link to document '../../readme.md'`.
2. Sometimes, in one of the `readme.md` files, I want to make a link that points to a directory. For example, the top-level `readme.md` has a link `[cosmos-config](cosmos-config/)`. It seems as though this ought to open netrw on that directory in neovim. Anyway, it works well in Bitbucket when I browse the markdown there. But in neovim, LSP gives an ERROR: `not found`.

**Comment by tevaughan** [comment]
I am seeing another error pop up now. In this case it has to do with the fact that not every note in my notebook is a markdown-file. I get an LSP-error ("not found") when I make a markdown-link from a markdown-file to a non-markdown file. (In this case, the file is source-code for gnuplot.)

**Comment by mickael-menu** [comment]
Number 1 seems like a bug. Would you mind opening an issue on [the `zk` repo](http://github.com/mickael-menu/zk) with a sample notebook reproducing the error?

Number 2 is expected. You are trying to use `zk-nvim` as a general Markdown LSP but it's really targeted at Markdown-based notebooks. However, [you can disable the error for `dead-link` diagnostics in the `zk` config](https://github.com/mickael-menu/zk/blob/main/docs/config-lsp.md#diagnostics).
